src/EventStore.Client/EventStore.Client.csproj

```diff
@@ -24,7 +24,7 @@
 		</AssemblyAttribute>
 	</ItemGroup>
 	<ItemGroup>
-		<Protobuf Include="..\EventStore.Client.Common\protos\shared.proto" GrpcServices="Client" ProtoRoot="..\EventStore.Client.Common\protos" />
+		<Protobuf Include="&#xA;			..\EventStore.Client.Common\protos\shared.proto;&#xA;			..\EventStore.Client.Common\protos\status.proto;&#xA;			..\EventStore.Client.Common\protos\code.proto;&#xA;		" GrpcServices="Client" ProtoRoot="..\EventStore.Client.Common\protos" />
```

Comment:
Is the `&#xA;` intentional?

Reply:
sigh. those are supposed to be carriage returns to make it easier to read. No such luck!